plugins/mcreator-core/datalists/blockstateproperties.yaml

```diff
@@ -0,0 +1,735 @@
+- ATTACHED:
+  readable_name: ATTACHED
+  type: Logic
+  other:
+    registry_name: attached
+- BERRIES:
+  readable_name: BERRIES
+  type: Logic
+  other:
+    registry_name: berries
+- BLOOM:
+  readable_name: BLOOM
+  type: Logic
+  other:
+    registry_name: bloom
+- BOTTOM:
+  readable_name: BOTTOM
+  type: Logic
+  other:
+    registry_name: bottom
+- CAN_SUMMON:
+  readable_name: CAN_SUMMON
+  type: Logic
+  other:
+    registry_name: can_summon
+- CONDITIONAL:
+  readable_name: CONDITIONAL
+  type: Logic
+  other:
+    registry_name: conditional
+- CRACKED:
+  readable_name: CRACKED
+  type: Logic
+  other:
+    registry_name: cracked
+- CRAFTING:
+  readable_name: CRAFTING
+  type: Logic
+  other:
+    registry_name: crafting
+- DISARMED:
+  readable_name: DISARMED
+  type: Logic
+  other:
+    registry_name: disarmed
+- DRAG:
+  readable_name: DRAG
+  type: Logic
+  other:
+    registry_name: drag
+- DUSTED:
+  readable_name: DUSTED
+  type: Logic

```

Comment:
To me DUSTED seems like integer property:

![image](https://github.com/user-attachments/assets/52975b77-6faa-4cdd-b609-116bc1d23a93)

Were mappings and data lists tested?

Reply:
Oh no, I lost this hide & seek :eyes: Thanks for catch